blog.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+    <title>Adopt A Dog</title>
+</head>
+<body>
+  <div class="grid-container">
+      <section id="banner">
+          <div class="filler">
+            </div>
+        <div id="letterhead">
+          <img src="images/adoptadog-logo.svg" id="toplogo" alt= "profile of a dog in white">
+          Seattle's premiere dog adoption agency
+        </div>
+        <div id="cart">
+          <div class="top-cart">
+            <img src="images/cart-icon.svg" id="carticon" alt="a white cart logo">
+            &nbsp;&nbsp;$1234.56
+          </div>
+          <div class="checkout-button">
+            Checkout
+          </div>
+        </div>
+        <div class="filler">
```

Comment:
not a fan of these filler divs. If you need things pushed in, let padding on the parent element handle it, or use grid or flexbox for layout